articles/install-guide/index.md

```diff
@@ -1,53 +1,207 @@
+   ```
 
-Then, check our [quickstart guide](xref:microsoft.quantum.write-program?tabs=tabid-vscode) for step-by-step instructions for your first quantum program.
+1. Verify the installation
 
+    - Create a minimal Q# operation, by creating a file called `Operation.qs`, and and adding the following code to it:
 
-### Python ###
+    ```qsharp
+    namespace HelloWorld
+    {
+        open Microsoft.Quantum.Intrinsic;
+        open Microsoft.Quantum.Canon;
 
-If you are a Python user, check out our [getting started with Python and the Quantum Development Kit guide](xref:microsoft.quantum.install.python).
+        operation HelloQ() : Result
+        {
+            Message($"Hello from quantum world!");
+            return Zero;
+        }
+    }
+    ```
 
+    - Create a python program called `run_hello_q.py` to call the Q# `HelloQ()` operation:
 
-## Learn Quantum Computing with Q# ##
+      ```python
+      from HelloWorld import HelloQ
 
-* [Quantum Katas](https://github.com/Microsoft/QuantumKatas)
-* [Samples](https://github.com/Microsoft/Quantum)
-* [Quantum Computing Concepts](xref:microsoft.quantum.concepts.intro)
-* [Microsoft Quantum Blog](https://cloudblogs.microsoft.com/quantum/?ext)
-* [Q# Blog](https://devblogs.microsoft.com/qsharp/)
+      HelloQ.simulate()
+      ```
 
+    - Run the program:
 
+    ```bash
+    python run_hello_q.py
+    ```
+
+    - Verify the output. Your program should output the following lines:
+
+    ```bash
+    Hello from quantum world!
+    0
+    ```
+
+## Develop with Jupyter notebooks
+
+1. Pre-requisites
+
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - [Jupyter notebooks](https://jupyter.readthedocs.io/en/latest/install.html)
+
+1. Install the `iqsharp` package:
+
+    ```bash
+    conda install -c quantum-engineering iqsharp
+    ```
+
+1. Verify the installation:
+
+    - Run the following command to start the notebook server:
+
+    ```bash
+    jupyter notebook
+    ```
+
+    - Browse to the URL shown on the command line. For example:
+
+    ```bash
+    http://localhost:8888
+    ```
+
+    - Create a jupyter notebook with a Q# kernel, and add the following code to the first notebook cell:
+
+    ```qsharp
+    operation HelloQ () : Unit {
+        Message($"Hello from quantum world!");
+    }
+    ```
+
+    - Run this cell of the notebook:
+
+    You should see `HelloQ` in the output of the cell. When running in jupyter notebooks, the Q# code is compiled, and the notebook outputs the name of the operation(s) that it finds.
+
+## Develop with C# on Windows, using Visual Studio
+
+1. Pre-requisites
+
+    - Visual Studio, version, with the .NET Core cross-platform development workload enabled
+
+1. Install the Q# Visual Studio extension
+
+    - Download the [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+    - Add the extension to Visual Studio
+
+1. Create a new Q# application
+
+1. Verify the installation
```

Comment:
I'm not sure what this means. What do I need to do to verify the installation? Is this just building the application?

Reply:
We should talk about this in the meeting? What are the minimal steps to verify that the installation has worked? And is there a better way to word this?